workflows/prognostic_c48_run/runtime/nudging.py

```diff
@@ -201,9 +201,15 @@ def _average_states(state_0: State, state_1: State, weight: float) -> State:
     for key in common_keys:
         if isinstance(state_1[key], xr.DataArray):
             with xr.set_options(keep_attrs=True):
-                out[key] = (
-                    state_0[key] * weight + (1 - weight) * state_1[key]  # type: ignore
-                )
+                if key != MASK:
```

Comment:
Since most of the variables (slsmk is the only one exception I can think of) are fine with using linear time interpolation, I made a 'blacklist' of the nearest neighbor interpolated vars since a mapping of all possible vars would break easily (e.g. a new variable was nudged). This is defined twice in both the nudging and prescriber modules since they use their own separate interpolation codes (didn't want to create a new tiny module to house a one item list).

Since this function is pretty low level and already just deals with just time interpolation, I felt ok leaving the if statement here- lmk if you feel otherwise. 

For the other instance in the prescriber I pulled the time interpolation out to its own function since it was a bit longer.